ecl/hql/hqlgram2.cpp

```diff
@@ -11958,16 +11985,15 @@ IHqlExpression *HqlGram::yyParse(bool _parsingTemplateAttribute, bool catchAbort
     }
     catch (IException * e)
     {
-        if (e->errorCode() != ERR_ABORT_PARSING)
+        if (catchAbort || e->errorCode() != ERR_ABORT_PARSING)
```

Comment:
Preferred change it to change it back to how it was, so the aborted event exception is not reported as an error.
If anywhere catches and reports it it should probably rethrow it instead.